infrastructure/transform/locals.tf

```diff
@@ -1,3 +1,12 @@
 locals {
-  storage_account_name = "dbfs${var.truncated_naming_suffix}"
+  storage_account_name    = "dbfs${var.truncated_naming_suffix}"
+  adb_linked_service_name = "ADBLinkedServiceViaMSI"
+
+  python_file_local_path = "../../transform/features"
+  python_file_dbfs_path  = "dbfs:"
```

Comment:
Sorry just seen this, but this seems the same as `whl_file_dbfs_path`, and also I guess it's not a path? How about calling it `dbfs_prefix` and removing `whl_file_dbfs_path`

Reply:
Will resolve just to get it merged as this will be superseded by some of my work anyway